eCommerce platforms/Adobe Magento/src/code/local/Microsoft/Dfp/etc/system.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Microsoft Dynamics 365 Fraud Protection module for Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Microsoft
+ * @package    Dynamics 365 Fraud Protection
+ * @copyright  Copyright (c) Microsoft Corporation. (https://www.microsoft.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+-->
+
+<config>
+  <tabs>
+    <microsoft translate="label" module="dfp">
+      <label>Microsoft</label>
+      <sort_order>10</sort_order>
+    </microsoft>
+  </tabs>
+  <sections>
+    <dfp translate="label" module="dfp">
+      <label>Dynamics 365 Fraud Protection</label>
+      <tab>microsoft</tab>
+      <frontend_type>text</frontend_type>
+      <sort_order>0</sort_order>
+      <show_in_default>1</show_in_default>
+      <show_in_website>1</show_in_website>
+      <show_in_store>1</show_in_store>
+      <groups>
+        <general translate="label">
+          <label>General Config</label>
+          <frontend_type>text</frontend_type>
+          <sort_order>0</sort_order>
+          <show_in_default>1</show_in_default>
+          <show_in_website>1</show_in_website>
+          <show_in_store>1</show_in_store>
+          <fields>
+            <assessmenttype translate="label">
+              <label>Purchase Assessment Type</label>
+              <comment>Select Evaluate if you are still evaluating Dynamics 365 Fraud Protection and not necessarily honoring its recommendations. Select Protect if you are honoring recommendations.</comment>
```

Comment:
Looking at the rest of the code, this actually changes the behavior of the module itself, right? If so, we should probably make it more explicit here that selecting Protect will accept/reject based on the recommendations.

Reply:
Great point.  I've updated the text.